builder/test/replica_pipeline.cc

```diff
@@ -549,4 +549,56 @@ auto p = []() -> ::slinky::pipeline {
   ASSERT_EQ(0, p().evaluate(inputs, outputs, eval_ctx));
 }
 
+TEST(replica, unused_output) {
+  // clang-format off
```

Comment:
It would be better to have a non-replica test for this, these replica tests are a bit annoying to maintain.

I think I'm also fine with no test in this case, the fix is simple enough and this case (unused output) doesn't necessarily need to work it just needs to be more obvious why it crashes.

Reply:
Removed the test